### ISSUE TITLE
Different link for specific galaxy instances of workflows and tutorials

### ISSUE THREAD

**Issue by paulzierep** [galaxyproject/training-material#4637]
The workflow and tutorial queries (https://training.galaxyproject.org/training-material/workflows/embed.html?query=microbiome) via the GTN are useful to filter for specific communities and are already used as iframes in subdomains such as https://singlecell.usegalaxy.eu/ and https://microgalaxy.usegalaxy.eu/:

The problem when using them in iframes for the subdomains is, that the links only point to the main instances i.e. usegalaxy.eu; this opens galaxy inside the iframe, it would be much more user-friendly if this could point to the actual instance i.e. https://microgalaxy.usegalaxy.eu/

**Comment by hexylena** [comment]
this should be doable, we could add a query parameter in which you pass the parent pages domain, and then do some replacement.

**Comment by paulzierep** [comment]
I thought something along those lines. It would probably also be good if we could make the links open in the parent frame, like `<a target="_parent" href="http://url.org">link</a>` ... taken from here: https://stackoverflow.com/questions/1037839/how-to-force-link-from-iframe-to-be-opened-in-the-parent-window

**Comment by hexylena** [comment]
yes that's likewise possible. A bit harder to know if we're in the right environment for that, but perhaps we can make it configurable or so.
